bl/policy/policy_factory.go

```diff
@@ -60,6 +68,40 @@ func CreatePolicy(policies *cliClient.EvaluationPrerunPolicies, policyName strin
 	return Policy{policyName, rules}, nil
 }
 
+func GetPoliciesFileFromPath(path string) (*cliClient.EvaluationPrerunPolicies, error) {
+	fileReader := fileReader.CreateFileReader(nil)
+	policiesStr, err := fileReader.ReadFileContent(path)
+	if err != nil {
+		return nil, err
+	}
+	validationResults, err := validatePoliciesYaml(policiesStr)
+	if err != nil {
+		return nil, err
+	}
+	if !validationResults.Valid() {
```

Comment:
This should be part of the validatepoliciesYaml function because then you have 2 places that are responsible for validation, the `validatePoliciesYaml` function and the if scope in this function

Reply:
done